### ISSUE TITLE
Icons - Initial testing for app

### ISSUE THREAD

**Issue by randomuser4444** [npwoods/bletchmame#230]
Hello, I wanted to create some initial icons if you are interested.
[Icons01.zip](https://github.com/npwoods/bletchmame/files/7490104/Icons01.zip)

Cheers,
Nerd4gw

**Comment by npwoods** [comment]
I was hoping that somebody would come along and create icons!

Its not clear to me how these are intended to be used.  My first read is that these are supposed to be:
* `Icon_Working_result.ico` is intended to be a background for icons for systems where audits have succeeded
* `Icon_Disable_result.ico` is intended to be a background for icons for systems where audits have failed
* `Icon_Unknown_result.ico` is intended to be a background for icons for systems where audits are pending

I'm not sure about the others

Also, It just feels like these are just random colors (yeah, green means success but still...).  At least the icons that are there derived from MAMEUI, while ugly and twenty years old, suggest what is happening

**Comment by randomuser4444** [comment]
Cool, these were created by me just for Bletch!

Although it's just a 'start' so we can probally do them properly later as these are only 64x64 one they are improved I can make the proper set with 32x32, 64x64, 96x96 128x128 etc.

The Icon_Main.ico is for optional for the main application. I thought the retro kinda colour with B would look clear and better than 'no' icon. But otherwise if you have an idea of what you like I can aim for that too.

Originally I was thinking these could replace the current icons if they are not available and if people put in an icon pack then those override but, just thinking now perhaps there could be another column and these are for status indication?

I put in a favourite icon for future when you want to mark a machine as favourite.

![image](https://user-images.githubusercontent.com/71576102/140628649-a9740f00-f944-4927-921d-826df82206f2.png)

**Comment by stavpup** [comment]
Is there support for zipped icon.zip ? It is a pain to unzip 33000+ files to a directory.

**Comment by npwoods** [comment]
@stavpup yes, you can add an Icons.zip to your icons path, similar to MAMEUI
